devicetypes/smartthings/zigbee-ceiling-fan-light.src/led-fan-lightings.groovy

```diff
@@ -124,6 +124,7 @@ private sendFanSpeed(val) {
 
 def ping() {
 	// PING is used by Device-Watch in attempt to reach the Device
+	zigbee.readAttribute(FAN_CLUSTER_VALUE, FAN_STATUS_VALUE)
```

Comment:
This won't do anything, as it isn't returned from the method.

You need something like
```
def ping() {
	return zigbee.onOffRefresh() +
		zigbee.readAttribute(FAN_CLUSTER_VALUE, FAN_STATUS_VALUE)
}
```
or
```
def ping() {
	zigbee.onOffRefresh() +
	zigbee.readAttribute(FAN_CLUSTER_VALUE, FAN_STATUS_VALUE)
}
```

Reply:
Thank you so much. I really appreciate it.